projects/ngneat/transloco-locale/src/lib/transloco-locale.config.ts

```diff
@@ -0,0 +1,42 @@
+import { InjectionToken } from '@angular/core';
+import { HashMap } from '@ngneat/transloco';
+import LOCALE_CURRENCY from './locale-currency.json';
+import { DateFormatOptions, NumberFormatOptions, Locale, Currency } from './transloco-locale.types';
+
+export interface LocaleSettings {
+  date?: DateFormatOptions;
+  decimal?: NumberFormatOptions;
+  currency?: NumberFormatOptions;
+  percent?: NumberFormatOptions;
+}
+
+export interface LocaleConfig {
+  global: LocaleSettings;
+  localeBased: HashMap<LocaleSettings>;
+}
+
+export interface TranslocoLocaleConfig {
+  number?: NumberFormatOptions;
+  date?: DateFormatOptions;
+  defaultLocale?: Locale;
+  localeConfig?: LocaleConfig;
```

Comment:
There is a weird mix with the usage of the words: settings and config. It should be aligned.